blog/2016-05-20-EclipseFuseTooling-NewGlobalConfiguration.adoc

```diff
@@ -0,0 +1,28 @@
+
+= JBoss Fuse Tooling - Support of Global configurations
+
+:page-date:2016-05-20
+:page-layout: blog
+:page-author: apupier
+:page-tags: [jbtis,fuse]
+
+Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/18/jboss-fuse-tooling-support-of-global-configurations/[Red Hat Developers Program blog]
+
+A new editor tab is now available to manipulate global configurations.
+

```

Comment:
IMO some introduction part is missing.
Something like you have in the original article in RHD blog:

> Red Hat JBoss Fuse is an open source, lightweight and modular integration platform that allows you to connect services and systems across your entire application portfolio. And if you’re familiar with Fuse, you’re probably familiar with the Fuse Tooling that comes with Red Hat JBoss Developer Studio.
> 
> As I mentioned in an earlier post, the 8.0.0.Beta2 version of JBoss Fuse Tooling is now available. Apart from the diagram tooling rework, there is yet another new, awaited feature. You can find it in the new “Configurations” editor — designed to manipulate global configurations, i.e. elements defined at the Camel context scope. 

Something about where this new feature available and how you can get it to try.